doc/rad_definitions.rst

```diff
@@ -263,23 +263,82 @@ In python it may look like this:
    >>> print([round(t, 8) for t in temp])
    [299.99998562, 301.00000518]
 
+This approach only works for monochromatic or very narrow bands for which the 
+spectral response function is assumed to be constant. In reality, typical imager
+channels are not that narrow and the spectral response function is not contstant 
+over the band. Here it 
+is not possible to un-concatenate planck and spectral response function. That 
```

Comment:
This needs better explanation. What is meant by
"un-concatenate planck and spectral response function"?